props/blaster.h

```diff
@@ -30,7 +35,7 @@ class Blaster : public PropBase {
     MODE_AUTO
   };
 
-  BlasterMode blaster_mode = MODE_STUN;
+  BlasterMode blaster_mode = MODE_KILL;
```

Comment:
This was not part of the PR description.

Reply:
Oops. Sorry. Ok do have it default start on kill?

Reply:
Let's not change it unless there is a reason to change it.